WORKSPACE

```diff
@@ -35,11 +35,11 @@ bazel_skylib_workspace()
 
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-git_repository(
+http_archive(
     name = "io_bazel_skydoc",
```

Comment:
Could you also rename this io_bazel_stardoc now? (If it's too much effort in this PR, feel free to defer

Reply:
Sure, done